content/id/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,184 @@
+
+## _Controller_ EndpointSlice
+
+_Controller_ EndpointSlice mengamati Service dan Pod untuk memastikan EndpointSlice
+yang bersangkutan berada dalam kondisi terkini. _Controller_ EndpointSlice akan mengelola EndpointSlice untuk
+setiap Service yang memiliki selektor. Ini akan merepresentasikan IP dari Pod
+yang cocok dengan selektor dari Service tersebut.
+
+### Ukuran EndpointSlice
+
+Secara bawaan, jumlah _endpoint_ yang dapat dimiliki tiap EndpointSlice dibatasi sebanyak 100 _endpoint_. Kamu dapat
+mengaturnya melalui opsi `--max-endpoints-per-slice` {{< glossary_tooltip
+text="kube-controller-manager" term_id="kube-controller-manager" >}} sampai dengan
+jumlah maksimum sebanyak 1000 _endpoint_.
+
+### Distribusi EndpointSlice
+
+Tiap EndpointSlice memiliki sekumpulan _port_ yang berlaku untuk seluruh _endpoint_ dalam sebuah sumber daya. Ketika nama _port_ digunakan untuk sebuah Service, Pod mungkin mendapatkan
+nomor target _port_ yang berbeda-beda untuk nama _port_ yang sama, sehingga membutuhkan
+EndpointSlice yang berbeda. Hal ini mirip dengan logika mengenai bagaimana _subset_ dikelompokkan
+dengan Endpoints.
+
+_Controller EndpointSlice_ akan mencoba untuk mengisi EndpointSlice sebanyak mungkin, tetapi tidak
+secara aktif melakukan _rebalance_ terhadap EndpointSlice tersebut. Logika dari _controller_ cukup sederhana:
+
+1. Melakukan iterasi terhadap EndpointSlice yang sudah ada, menghapus _endpoint_ yang sudah tidak lagi
+   dibutuhkan dan memperbarui _endpoint_ yang sesuai yang mungkin telah berubah.
+2. Melakukan iterasi terhadap EndpointSlice yang sudah dimodifikasi pada langkah pertama dan
+   mengisinya dengan _endpoint_ baru yang dibutuhkan.
+3. Jika masih tersisa _endpoint_ baru untuk ditambahkan, mencoba untuk menambahkannya pada
+   _slice_ yang tidak berubah sebelumnya dan/atau membuat _slice_ yang baru.
+   
+Terlebih penting, langkah ketiga memprioritaskan untuk membatasi pembaruan EndpointSlice terhadap
+distribusi dari EndpointSlice yang benar-benar penuh. Sebagai contoh, jika ada 10
+_endpoint_ baru untuk ditambahkan dan ada 2 EndpointSlice yang masing-masing memiliki ruang untuk 5 _endpoint_ baru,
+pendekatan ini akan membuat sebuah EndpointSlice baru daripada mengisi 2
+EndpointSlice yang sudah ada. Dengan kata lain, pembuatan sebuah EndpointSlice 
+lebih diutamakan daripada pembaruan beberapa EndpointSlice.
+
+Dengan kube-proxy yang berjalan pada tiap Node dan mengamati EndpointSlice, setiap perubahan
+pada sebuah EndpointSlice menjadi sangat mahal karena hal tersebut akan dikirimkan ke
+setiap Node dalam klaster. Pendekatan ini ditujukan untuk membatasi jumlah
+perubahan yang perlu dikirimkan ke setiap Node, meskipun hal tersebut berdampak pada banyaknya
+EndpointSlice yang tidak penuh.
+
+Pada praktiknya, distribusi yang kurang ideal seperti ini akan jarang ditemukan. Kebanyakan perubahan yang diproses oleh _controller_ EndpointSlice akan sangat kecil untuk dapat masuk pada
```

Comment:
```suggestion
Pada praktiknya, distribusi yang kurang ideal seperti ini akan jarang ditemukan. Kebanyakan perubahan yang diproses oleh _controller_ EndpointSlice akan cukup kecil untuk dapat masuk pada
```

An important translation. "Sangat kecil" has the feel of "too small" in this context.
The original doc says "small enough"